Dockerfile

```diff
@@ -14,19 +15,17 @@ COPY . /src/panacea-core
 # Set working directory for the 'panacea-core' build
 WORKDIR /src/panacea-core
 
-# Install panacea-core
-RUN make clean && make build
-
 # Get 'libwasmvm.so' from wasmvm
-RUN git clone -b v0.14.0 https://github.com/CosmWasm/wasmvm.git /src/wasmvm
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v0.16.2/libwasmvm_muslc.a /lib/libwasmvm_muslc.a
+RUN sha256sum /lib/libwasmvm_muslc.a | grep 0e62296b9f24cf3a05f8513f99cee536c7087079855ea6ffb4f89b35eccdaa66
+
+RUN make clean && LEDGER_ENABLED=true BUILD_TAGS=muslc make build
```

Comment:
Actually, we don't need to set the `LEDGER_ENABLED` env var :)

In the `Makefile`, we always enable the `ledger` build tag.
https://github.com/medibloc/panacea-core/blob/e46b2d6f72569e57141a7f1c806a1b82fa33da15/Makefile#L15

But, as you did, the `BUILD_TAGS=muslc` must be set.

Reply:
Thanks for your feedback!

I change wasmvm version and checksum data.

Also I removed `LEDGER_ENABLED` env var

Thank you :)